### PR TITLE
Add unstyled election filter.

### PR DESCRIPTION
**Pull request by jmcarp**
Needs revisions and tests--propose to defer until after usability testing.

cc @noahmanger

### CODE REVIEW

**Comment by noahmanger** [comment]
+1 to tests coming after usability testing. As can style refinements.

@ccostino do you mind taking a look at this? I think best to have you as the primary reviewer on JS from here on out.

**Comment by ccostino** [comment]
Sure thing!  That sounds good to me too with the JS reviews, I'm happy to do that.

**Comment by ccostino** [comment]
Let me know when I should take a closer look.  As a start, this looks to be sound so far.

File: js/filters.js [hunk 1]
```diff
@@ -177,4 +183,73 @@ TypeaheadFilter.prototype.handleNestedChange = function(e) {
...
+  }.bind(this));
+  bins.unshift({
+    name: this.name,
+    cycle: election,
+    min: election - this.duration + 1,
+    max: election,
+    full: true
+  });
+  this.$cycles.html(cyclesTemplate(bins));
+  this.$cycles.find('input').eq(0).prop('selected', true).change();
```
Comment (noahmanger) [inline js/filters.js]: Small thing, but in other toggles we use the attribute "checked". Not sure if you care. Looks like it does the same thing.

File: js/templates/election-cycles.hbs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{{#each this}}
+  <label>
```
Comment (noahmanger) [inline js/templates/election-cycles.hbs]: Label should have a `for=[id of input]` attribute and then the `input` needs the same id.

**Comment by noahmanger** [comment]
Apart from my small notes, two behavior questions:

1. When the select is first changed, what is the default time period? Is it the full cycle? If so, it doesn't look like that gets its `selected`/`checked` property set.
2. Do you want to add the filter tags functionality in a subsequent patch or include it in here? I'm ok with either.

**Comment by jmcarp** [comment]
@noahmanger I updated the markup and fixed selections on election year change. Seems reasonable to implement filter tags and styling in separate patches.

**Comment by noahmanger** [comment]
:+1: I'll make a new issue for adding the tags.

**Pull request merged**